common/buildcraft/silicon/render/PackageFontRenderer.java

```diff
@@ -33,6 +34,7 @@ public int getStringWidth(String s) {
 
 	@Override
 	public int drawString(String s, int x, int y, int color, boolean shadow) {
+		s = EnumChatFormatting.getTextWithoutFormattingCodes(s);
```

Comment:
NEI shouldn't be affecting these lines, but that's a tolerable patch; either way, this should run *after* the `realRenderer.drawString` check/call, because otherwise it will strip formatting codes from lines which are not affected by the code below.